tensorboard/webapp/metrics/store/metrics_selectors_test.ts

```diff
@@ -1298,6 +1298,18 @@ describe('metrics selectors', () => {
       );
       expect(selectors.getMetricsCardMinWidth(state)).toBe(400);
     });
+
+    it('returns savingPinsEnabled when called getMetricsSavingPinsEnabled', () => {
```

Comment:
nit: the test isn't very robust, as many different expressions could evaluate to `false`. A easy, succinct way to test both true/false can be [parameterized testing](https://github.com/tensorflow/tensorboard/pull/6725#discussion_r1461316740)